<launch>
  <!-- Launch rviz.launch to visualize this demo -->

  <!-- Debug -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <include file="$(find panda_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <group ns="grasp_poses_visualizer">
    <!-- Which arm to use -->
    <param name="ee_group_name" value="hand"/>
    <param name="planning_group_name" value="panda_arm"/>
    <rosparam command="load" file="$(find moveit_grasps)/config_robot/panda_grasp_data.yaml"/>
  </group>

  <!-- Start the demo -->
  <node name="grasp_poses_visualizer_demo" launch-prefix="$(arg launch_prefix)" pkg="moveit_grasps"
	type="grasp_poses_visualizer_demo" output="screen" args="">
    <rosparam command="load" file="$(find moveit_grasps)/config/moveit_grasps_config.yaml"/>
  </node>

</launch>
